worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/implementation/blocks/CharBlocks.java

```diff
@@ -70,10 +70,7 @@ public boolean isFull() {
     };
     public char[][] blocks;
     public Section[] sections;
-    public Object[] sectionLocks;
-    protected int minSectionPosition;
-    protected int maxSectionPosition;
-    protected int sectionCount;
+
```

Comment:
remove added blank line

Reply:
Removed the blank lines in my latest commit!